playbook-website/app/javascript/site_styles/map_styles/_map_default.scss

```diff
@@ -0,0 +1,12 @@
+@import "playbook-ui/dist/tokens/colors";
+@import "playbook-ui/dist/tokens/spacing";
+@import "playbook-ui/dist/tokens/shadows";
+@import url("https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.css");
+@import url("https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-draw/v1.3.0/mapbox-gl-draw.css");
+
+
+.pb_map {
+  height: 600px;
```

Comment:
I just wonder how this should function on mobile/responsive views... 
Height at 600px always is pretty strict. Any thoughts? @nidaqg

Reply:
The library needs a height passed to it for the map to render. we can also set a height to the parent div and set this height to 100%,  I can try it with max-height if you want